### DISCUSSION TITLE
IOS-XR 6.6.3 get_bgp_neighbors_detail(neighbor_address='') returns all entries

### DISCUSSION THREAD

**Discussion by Mondhund** [napalm-automation/napalm#1405] [Q&A]
(using python3, napalm 3.2.0)
as said: a call to e.g. get_bgp_neighbors_detail('192.168.0.1') always returns details about all neighbors on the router instead of just one. 
In addition I get messages

>  'list' object has no attribute 'xpath'

. 
It makes no difference what's in the brackets, e.g. ('lksldfl') or (neighbor_address='192.168.0.1') always returns the same output.

Thanks!

**Comment by mirceaulinic** [comment]
Hey @Mondhund - this looks like a bug. Would you open an issue with all the required details?

**Reply by Mondhund** [reply]
Hi @mirceaulinic, thanks - will do so.
